site/pages/faq.html

```diff
@@ -0,0 +1,45 @@
+---
+title: FAQ
+description: FAQ on the FT's engineering competencies
+permalink: /docs/faq
+layout: o-layout-docs
+---
+
+# {{page.title}}
+
+## Is there an individual contributor route?
+
+There is no separate individual contributor route to promotion, but the aim is
+to have a single set of competencies that are flexible enough to allow for a
+variety of ways that an engineer can demonstrate seniority.
+
+A key idea here is that the more senior you are as an engineer, the more you
+should be helping those around you to achieve. There are many ways to achieve
```

Comment:
Slight jar with repeated "achieve". Suggestion:
```suggestion
should be helping those around you to achieve. There are many ways to do
```